dist/intern.d.ts

```diff
@@ -1,7 +1,7 @@
 declare const intern: {
     findKnex(knex: any, ent: any, q: any): Promise<any>;
     firstKnex(knex: any, ent: any, q: any): Promise<any>;
-    insertKnex(knex: any, ent: any, q: any): Promise<any>;
+    insertKnex(knex: any, ent: any): Promise<any>;
```

Comment:
Knex has types: https://github.com/knex/knex/blob/master/types/index.d.ts

Because we are using TypeScript, let's use those types for the `knex` argument and the return value.

This request applies to all functions that make use of `knex`.

Reply:
Done using :

`import type { Knex } from 'knex';`